lib/HLSL/HLMatrixLowerPass.cpp

```diff
@@ -411,29 +411,29 @@ INITIALIZE_PASS(HLMatrixLowerPass, "hlmatrixlower", "HLSL High-Level Matrix Lowe
 
 static Instruction *CreateTypeCast(HLCastOpcode castOp, Type *toTy, Value *src,
```

Comment:
This is a minor duplication of ConvertScalarOrVector in CGHLSLMS.cpp, unfortunately. But because of the different IRBuilder template specializations, I can't move it to HLModule.h